controller/jobs.go

```diff
@@ -102,20 +110,27 @@ func (r *JobRepo) Add(job *ct.Job) error {
 
 func scanJob(s postgres.Scanner) (*ct.Job, error) {
 	job := &ct.Job{}
-	err := s.Scan(&job.ID, &job.AppID, &job.ReleaseID, &job.Type, &job.State, &job.CreatedAt, &job.UpdatedAt)
+	var meta hstore.Hstore
+	err := s.Scan(&job.ID, &job.AppID, &job.ReleaseID, &job.Type, &job.State, &meta, &job.CreatedAt, &job.UpdatedAt)
 	if err != nil {
 		if err == sql.ErrNoRows {
 			err = ErrNotFound
 		}
 		return nil, err
 	}
+	if len(meta.Map) > 0 {
+		job.Meta = make(map[string]string, len(meta.Map))
+		for k, v := range meta.Map {
+			job.Meta[k] = v.String
+		}
```

Comment:
I don't think we do as this function is called for both Get and List. The opposite direction could share a helper function with `AppRepo.Add` for creating an `hstore.Hstore` from `map[string]string`.